config.toml

```diff
@@ -428,7 +428,7 @@ language_alternatives = ["en"]
 [languages.hi]
 title = "Kubernetes"
 description = "प्रोडक्शन-ग्रेड कंटेनर ऑर्केस्ट्रेशन"
-languageName = "हिन्दी"
+languageName = "हिन्दी Hindi"
```

Comment:
This doesn't look right - the other localizations don't have their name in two alphabets / languages.

Reply:
I think Chinese, Korean and Japanese teams also do this since these languages use their own unique script like Hindi. So I think its better to write it in both Latin and its own script (devnagri in case of Hindi), isn't it?

Reply:
It helps folks who can't read these scripts know which language it is when they click drop down menu on the website

Reply:
Ah, good point; I hadn't considered the wider context. Chinese, Japanese and Korean used block-based writing systems that have an ideographic heritage, whereas our other localizations use alphabets or similar. I can see a distinction there but maybe it's not a helpful one.

Russian and Ukrainian uses an alphabet; those cases are similar to Hindi (several languages can be written using devanagari, not just Hindi).

I suggest taking this to the localization subproject to decide how we want to handle this. For now, I'd leave it as it is (matching other alphabets, abugidas etc), pending that discussion and its outcome.

Reply:
I believe Korean Hangul writing system is a phonogram (Although it did generated from ideograms). Yeah, many languages from Indian subcontinent can be written in Devnagri (especially those generating from Sanskrit).

Thankyou for bringing this up, We will take this to localization subproject.